### DISCUSSION TITLE
[Feature] Merge faces (mobile)

### DISCUSSION THREAD

**Discussion by Chuckame** [immich-app/immich#6259] [Feature Request]
### The feature

Would be great to merge faces using the mobile (only accessible through webapp). When importing the library for the first time, this task of naming/merging people can take so much time, so having the feature on mobile would improve the user experience.

_I did not find it in issues or discussions, sorry if it was already asked_

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by need4swede** [comment]
Any updates on this? Would be nice to be able to merge people through the mobile app

**Comment by codecrafting-io** [comment]
Yeah, also would be good to change the main photo for the face as you can on the Web version. Really hopeful for these features on mobile

**Comment by Vigno04** [comment]
Since now we are at the official release this feature is very inportant, because now there issn't parity between the web and the app

**Comment by bo0tzz** [comment]
Will be closed by #22670

**Reply by manuelkue** [reply]
The PR has been closed due to inactivity. Therefore, this feature request is still open and it seems nobody is working on it now.

**Comment by mw00** [comment]
This feature is not available on the android app and I can't find anywhere it is being worked on.

**Comment by YarosMallorca** [comment]
I've opened a PR for it! #30193
